doc/nimc.rst

```diff
@@ -437,10 +437,10 @@ Define                   Effect
 ``ssl``                  Enables OpenSSL support for the sockets module.
 ``memProfiler``          Enables memory profiling for the native GC.
 ``uClibc``               Use uClibc instead of libc. (Relevant for Unix-like OSes)
-``checkAbi``             When using types from C headers, add checks that compare
+``checkAbi`` (obsolete)  When using types from C headers, add checks that compare
                          what's in the Nim file with what's in the C header
                          (requires a C compiler with _Static_assert support, like
-                         any C11 compiler)
+                         any C11 compiler). Obsolete: now enabled by default.
```

Comment:
I would not declare it as obsolete here. But mentioning that it is enabled by default, I approve.

Reply:
re-worded and made -d:checkAbi non-default again because of pre-existing issues encountered in this PR that can absolutely be fixed in subsequent PR's instead (eg so that this PR introduces 0 breaking change)